janusgraph-core/src/main/java/org/janusgraph/graphdb/transaction/addedrelations/AddedPropertiesSetValue.java

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 Unified Catalog Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.graphdb.transaction.addedrelations;
+
+import org.apache.tinkerpop.gremlin.structure.Property;
+import org.janusgraph.core.JanusGraphVertexProperty;
+import org.janusgraph.graphdb.internal.InternalRelation;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class AddedPropertiesSetValue implements AddedPropertiesValue {
+
+    private final Map<Object, InternalRelation> internalRelationMap;
+
+    public AddedPropertiesSetValue() {
+        this.internalRelationMap = new HashMap<>();
+    }
+
+    @Override
+    public int addValue(InternalRelation internalRelation) {
+        int sizeBefore = this.internalRelationMap.size();
+        JanusGraphVertexProperty p = (JanusGraphVertexProperty) internalRelation;
+        Property prop = (Property) internalRelation;
+        this.internalRelationMap.put(prop.value(), internalRelation);
+        return this.internalRelationMap.size() - sizeBefore;
```

Comment:
As Codacy says it seems `p` isn't used anywhere. Instead you use `prop` below. You can simply remove this line: `JanusGraphVertexProperty p = (JanusGraphVertexProperty) internalRelation;`

Reply:
Done